src/pages/app-development/starter-kit/developer-tools.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Developer tools
+description: Learn how to use environment variables, logging, and testing with the Starter Kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+
+# Developer tools
+
+The following sections describe development tools and processes available in the Starter Kit.
+
+## Parameters as environment variables
+
+The `actionparams` object allows you to pass values from your environment variables to an action. Add the parameter to your `.env` file:
+
+`YOUR_PARAM=your_value`
+
+To pass the parameters to the action, add the parameter as an `input` under the corresponding action in the `actions/{entity}/../actions.config.yaml` file:
+
+```yaml
+{action name}:
+  function: commerce/{action name}/index.js
+  web: 'no'
+  runtime: nodejs:16
+  inputs:
+    LOG_LEVEL: debug
+    YOUR_PARAM: $YOUR_PARAM_ENV
```

Comment:
Previously the param in the `.env` file has been defined as `YOUR_PARAM` in line 22. For consistency, either:
- change this line to 
```YOUR_PARAM: $YOUR_PARAM```, or
- change line 22 to
```YOUR_PARAM_ENV=your_value```